docs/parser.md

```diff
@@ -172,43 +182,57 @@ require("@babel/parser").parse("code", {
+| `v7.11.0` | Added `decimal` |
+| `v7.10.0` | Added `privateIn` |
 | `v7.7.0` | Added `topLevelAwait` |
 | `v7.4.0` | Added `partialApplication` |
+| `v7.2.0` | Added `classPrivateMethods` |
 </details>
 
 | Name | Code Example |
 |------|--------------|
-| `asyncGenerators` ([proposal](https://github.com/tc39/proposal-async-iteration)) | `async function*() {}`, `for await (let a of b) {}` |
-| `bigInt` ([proposal](https://github.com/tc39/proposal-bigint)) | `100n` |
 | `classProperties` ([proposal](https://github.com/tc39/proposal-class-public-fields)) | `class A { b = 1; }` |
 | `classPrivateProperties` ([proposal](https://github.com/tc39/proposal-private-fields)) | `class A { #b = 1; }` |
 | `classPrivateMethods` ([proposal](https://github.com/tc39/proposal-private-methods)) | `class A { #c() {} }` |
 | `classStaticBlock` ([proposal](https://github.com/tc39/proposal-class-static-block)) | `class A { static {} }` |
+| `decimal` ([proposal](https://github.com/tc39/proposal-decimal)) | `0.3m` |
 | `decorators` ([proposal](https://github.com/tc39/proposal-decorators)) <br> `decorators-legacy` | `@a class A {}` |
 | `doExpressions` ([proposal](https://github.com/tc39/proposal-do-expressions)) | `var a = do { if (true) { 'hi'; } };` |
-| `dynamicImport` ([proposal](https://github.com/tc39/proposal-dynamic-import)) | `import('./guy').then(a)` |
-| `exportDefaultFrom` ([proposal](https://github.com/leebyron/ecmascript-export-default-from)) | `export v from "mod"` |
-| `exportNamespaceFrom` ([proposal](https://github.com/leebyron/ecmascript-export-ns-from)) | `export * as ns from "mod"` |
+| `exportDefaultFrom` ([proposal](https://github.com/tc39/ecmascript-export-default-from)) | `export v from "mod"` |
 | `functionBind` ([proposal](https://github.com/zenparsing/es-function-bind)) | `a::b`, `::console.log` |
-| `functionSent` | `function.sent` |
 | `importAssertions` ([proposal](https://github.com/tc39/proposal-import-assertions)) | `import json from "./foo.json" assert { type: "json" };` |
-| `importMeta` ([proposal](https://github.com/tc39/proposal-import-meta)) | `import.meta.url` |
+| `moduleStringNames` ([proposal](https://github.com/tc39/ecma262/pull/2154)) | `import { "😄" as smile } from "emoji";` |
+| `partialApplication` ([proposal](https://github.com/babel/proposals/issues/32)) | `f(?, a)` |
+| `pipelineOperator` ([proposal](https://github.com/babel/proposals/issues/29)) | <code>a &#124;> b</code> |
+| `privateIn` ([proposal](https://github.com/tc39/proposal-private-fields-in-in)) | `#p in obj` |
+| `throwExpressions` ([proposal](https://github.com/babel/proposals/issues/23)) | `() => throw new Error("")` |
+| `topLevelAwait` ([proposal](https://github.com/tc39/proposal-top-level-await/)) | `await promise` in modules |
+
+#### Latest ECMAScript features
+
+These features are already enabled on latest `@babel/parser`, they can not be disabled because they are part of the language.
```

Comment:
```suggestion
The following features are already enabled on the latest version of `@babel/parser`, and cannot be disabled because they are part of the language.
```